vars/deployPlannerSnapshot.groovy

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/groovy
+
+import java.util.regex.Pattern
+
+def call(body) {
+    // evaluate the body block, and collect configuration into the object
+    def config = [:]
+    body.resolveStrategy = Closure.DELEGATE_FIRST
+    body.delegate = config
+    body()
+
+    def yaml
+    def openShiftProject = config.openShiftProject
+    def openShiftTemplate = config.openShiftTemplate
+    def originalImageName = config.originalImageName
+    def newImageName = config.newImageName
+    def deploymentName = config.githubRepo
+    def providerLabel = config.providerLabel ?: 'fabric8'
+    def project = config.githubProject
+
+    def flow = new io.fabric8.Fabric8Commands()
+    def utils = new io.fabric8.Utils()
+
+    openShiftProject = openShiftProject + '-' + utils.getRepoName()
+    container('clients') {
```

Comment:
Spawning the [clients](https://github.com/fabric8io-images/builder-clients) container just to get oc binary seems a bit too much. Should probably add the 'oc' binary to the jenkins builder image and scrap this. Need not be part of this PR obviously, but something that needs a look. WDTY @rupalibehera @chmouel @sthaha

Reply:
I am not sure what you meant by `jenkins builder image`; not an area I know. is it the jenkins master image?

Reply:
@sthaha  Yes. I meant master. Looks like we bring in yet another repository and image and container just for the oc binary from https://github.com/fabric8io-images/builder-clients, which is what I was hoping to get rid of.